src/theme/palette.ts

```diff
@@ -377,6 +385,9 @@ export const darkModePalette: PaletteOptions = {
     },
     surfaces: Colors.accentGrey[10]
   },
+  component: {
+    tabs: Colors.charcoal[70]
```

Comment:
@SAHU-01 i think you miss in any one the mode to put different color, both are here same.

Reply:
apologies it's charcoal[110] sorry will update it.